pycode/memilio-epidata/memilio/epidata_test/test_epidata_getHospitalizationData.py

```diff
@@ -28,12 +28,12 @@
 
 from memilio.epidata import getDataIntoPandasDataFrame as gd
 from memilio.epidata import getHospitalizationData as ghd
-from memilio.epidata import progress_indicator
 
 
 class TestGetHospitalizationData(fake_filesystem_unittest.TestCase):
 
     maxDiff = None
+    gd.Conf.v_level = 'Debug'
```

Comment:
Same as above: why do you set the verbosity level to Debug?

Reply:
Same as above, except that the sanity_check does not have keywordarguments, so that the v_level is changed via gd.Conf.v_level.